crates/compiler/test_gen/src/helpers/wasm.rs

```diff
@@ -376,7 +378,7 @@ macro_rules! assert_evals_to {
         let phantom = std::marker::PhantomData;
         let _ = $ignore_problems; // Always ignore "problems"! One backend (LLVM) is enough to cover them.
         match $crate::helpers::wasm::assert_evals_to_help::<$ty>($src, phantom) {
-            Err(msg) => panic!("{}", msg),
```

Comment:
Same comment as above

Reply:
fixed in https://github.com/roc-lang/roc/compare/100a2df04327f19680e578c0f1bb2b87355b4311..e5eb5717eace1559d73561099e9381630b497634